modules/end-user-guide/pages/git-credential-store.adoc

```diff
@@ -1,25 +1,89 @@
-:navtitle: Git credential store
+:navtitle: Creating a {orch-name} Secret for a Git credentials store
 :keywords: user-guide, configuring, user, secrets
 :page-aliases: 
 
-[id="mounting-a-git-credential-store-into-a-workspace-container_{context}"]
-= Mounting a Git credentials store into a workspace container
+[id="mounting-a-git-credential-store-into-workspace-containers_{context}"]
+= Creating a {orch-name} Secret for a Git credentials store
 
-This section describes how to mount Git credentials store as secret from the user's {orch-namespace} into the file in single-workspace or multiple-workspace containers of {prod-short}.
+This section describes how to create a Secret containing a Git credentials store in the user's {orch-namespace}. The Git credentials store specified in the Secret is mounted as a file in the {devworkspace}'s containers.
```

Comment:
```suggestion
As an alternative to the OAuth for GitHub, GitLab, or Bitbucket that is configured by the administrator of your organization's {prod-short} instance, you can mount your Git credentials store as a Secret in your user {orch-namespace}. The Git credentials store specified in the Secret is mounted as a file in the {devworkspace} containers.
```

Reply:
Also, I've removed the last sentence `This Secret is mounted as a file in the {devworkspace} containers. ` because this info seems to be restated in the next paragraph.

Reply:
In my suggestion above, I've removed the last sentence This Secret is mounted as a file in the {devworkspace} containers.  because this info seems to be restated in the next paragraph.